modules/orchestrator/proc-install-components-for-orchestrator-plugin.adoc

```diff
@@ -0,0 +1,74 @@
+:_mod-docs-content-type: PROCEDURE
+[id="proc-install-components-for-orchestrator-plugin_{context}"]
+= Installing the components for the Orchestrator plugin on {ocp-short}
+
+To run the Orchestrator plugin successfully on {ocp-short}, you must install components that provide the runtime environment and the dependencies that the plugin requires.
+
+.Prerequisites
+
+* To install the Orchestrator plugin on {ocp-short}, you require the following components installed:
+
+** OpenShift Serverless Operator
+** Knative Serving
+** Knative Eventing
+** OpenShift Serverless Logic Operator
++
+[NOTE]
+====
+The {product} Operator provisions these components automatically. Alternatively, you can install them manually by using another Operator.
+====
```

Comment:
@jmagak As discussed RHDH Operator does not provide these infrastructural component automatically.
I'd recommend just remove this "The {product} Operator provisions these components automatically. Alternatively, you can install them manually by using another Operator."